superset-frontend/src/SqlLab/components/ShareSqlLabQuery.tsx

```diff
@@ -114,36 +115,33 @@ class ShareSqlLabQuery extends React.Component {
         </Button>
       </Styles>
     );
-  }
+  };
 
-  render() {
-    const canShare =
-      this.props.queryEditor.remoteId ||
-      isFeatureEnabled(FeatureFlag.SHARE_QUERIES_VIA_KV_STORE);
-    return (
-      <Tooltip
-        id="copy_link"
-        placement="top"
-        title={
-          canShare
-            ? t('Copy query link to your clipboard')
-            : t('Save the query to copy the link')
-        }
-      >
-        {canShare ? (
-          <CopyToClipboard
-            getText={callback => this.getCopyUrl(callback)}
-            wrapped={false}
-            copyNode={this.buildButton()}
-          />
-        ) : (
-          this.buildButton()
-        )}
-      </Tooltip>
-    );
-  }
-}
+  const canShare =
+    queryEditor.remoteId ||
+    isFeatureEnabled(FeatureFlag.SHARE_QUERIES_VIA_KV_STORE);
 
-ShareSqlLabQuery.propTypes = propTypes;
+  return (
+    <Tooltip
+      id="copy_link"
+      placement="top"
+      title={
+        canShare
+          ? t('Copy query link to your clipboard')
+          : t('Save the query to copy the link')
+      }
+    >
+      {canShare ? (
+        <CopyToClipboard
+          getText={(callback: any) => getCopyUrl(callback)}
```

Comment:
this was existing code, but since we're just passing the arg right on through, we should be able to just pass the function to the prop, like `getText={getCopyUrl}`